### PR TITLE
Improve kyverno docs

### PR DESCRIPTION
**Pull request by Poor12**
Signed-off-by: Poor12 <shentiecheng@huawei.com>

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Reform the docs and note that release-1.7 does not work for Karmada.

**Which issue(s) this PR fixes**:
Fixes https://github.com/karmada-io/karmada/issues/2459

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/userguide/security-governance/working-with-kyverno.md [hunk 1]
```diff
@@ -4,313 +4,350 @@ title: Working with Kyverno
...
-```console
+```sh
+git clone https://github.com/karmada-io/karmada
+cd karmada
 hack/local-up-karmada.sh
 ```
 
 ## Kyverno Installations
 
 In this case, we will use Kyverno v1.6.3. Related deployment files are from [here](https://github.com/kyverno/kyverno/blob/release-1.6/config/install.yaml).
```
Comment (RainbowMango) [inline docs/userguide/security-governance/working-with-kyverno.md:22]: still use v1.6.3?

Comment (Poor12) [reply docs/userguide/security-governance/working-with-kyverno.md:22]: The changes are the same for different versions. Not sure we should update docs if kyverno releases a new version.

Comment (RainbowMango) [reply docs/userguide/security-governance/working-with-kyverno.md:22]: We don't need to test and update the doc for each kyverno release. 
But we need to describe the correct version that was used to write this doc.

Comment (Poor12) [reply docs/userguide/security-governance/working-with-kyverno.md:22]: Yes, the following Yaml is an example given based on version 1.6.3. The version is correct with the corresponding Yaml file. I think not need to update all of them to 1.8.0.

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**